### ISSUE TITLE
Unable to add configs

### ISSUE THREAD

**Issue by Mnky313** [Kira-NT/packed-inventory#6]
I'm trying to allow other blocks with GUIs to be used by adding them in the config, for this example I'll use charm:woodcutter.

If I add it to the config manually or with ModMenu and restart the game it will not display in the ModMenu settings pane and will not work. 
![image](https://user-images.githubusercontent.com/23646586/194463751-cf16025e-6b5d-4239-9cd5-8b039228fc13.png)
Changing any setting from ModMenu will then delete all the options added from the config (adding test overwrote charm:woodcutter):
![image](https://user-images.githubusercontent.com/23646586/194463851-c01ee081-35a6-4247-9308-627fe3979191.png)

(updated title because it doesn't work with other vanilla stuff like minecraft:enchanting_table)

**Comment by Kira-NT** [comment]
**TL;DR** - this is the expected behavior

------

Let me explain, how this works. Configs are meant to be read by something, and if nothing reads them, they are just useless and therefore should be removed in order to not clog the config file.

There is no such abstraction in Minecraft as a "workstation" (despite some obviously shared functionality), so it's impossible to add support for a generic workstation that could handle crafting tables, anvils, `charm:woodcutter`s, and so on at once. A programmer must register a custom handler for each one of them manually, this is not something you are able to do via config. A custom config entry is part of a custom handler, not a replacement.

Therefore, if you want to be able to open a `charm:woodcutter` from your inventory, you need to kindly ask developers of the mod that adds this block to register it via Packed Inventory's API. I'm happy to help them in achieving it if something in README/examples is unclear.

Speaking of an enchantment table, to be quite frankly honest with you, I completely and utterly forgot that this thing still exists in Minecraft, it's kinda obsolete nowadays with all these "new" enchanting techniques. I'll try to implement a built-in support for it too, but no promises, since this is a block with a high level of awareness of its environment, so it may be tricky to supply a player with the enchanting GUI without a physical block being present in the world.
